runtime/gc_base/RootScanner.hpp

```diff
@@ -435,6 +435,7 @@ class MM_RootScanner : public MM_BaseVirtual
 	 */
 	virtual void scanOwnableSynchronizerObjects(MM_EnvironmentBase *env);
 	virtual void scanContinuationObjects(MM_EnvironmentBase *env);
+	virtual void iterateAllContinuationObjects(MM_EnvironmentBase *env);
```

Comment:
put some comments in this base RootScanner class for these 2 methods: that one is about pruning C.O. lists  within a scope of current GC (for example just for Nursery in Scavenge) and the other one is about iterating all live COs  in the heap and doing any processing that other parties may register through a hook.